packages/components/forms/src/base-input/BaseInput.styles.ts

```diff
@@ -1,7 +1,26 @@
 import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
+import type { CSSObject } from '@emotion/serialize';
 
-const getStyles = ({ as, isDisabled, isInvalid }) => ({
+const sizeToStyles = ({ size }): CSSObject => {
+  switch (size) {
+    case 'small':
+      return {
+        fontSize: tokens.fontSizeM,
+        lineHeight: tokens.lineHeightM,
+        padding: `${tokens.spacing2Xs} ${tokens.spacingXs}`,
+        height: `32px`,
```

Comment:
This is what I meant in the other comment

In `padding`, the use of the backtick makes sense because you are passing the tokens to the string
but in `height` you don't, so it should be just a single quote